cms/config/sync/user-role.public.json

```diff
@@ -4,89 +4,17 @@
   "type": "public",
   "permissions": [
     {
-      "action": "api::data-info.data-info.find"
```

Comment:
This change removed permissions for the FE to most of the data endpoints - please be careful when merging changes to content types, ideally separate them from other changes to make it easier to spot such problems.